conan/tools/build/flags.py

```diff
@@ -165,10 +165,15 @@ def build_type_flags(settings):
     return []
 
 
-def cppstd_flag(settings):
-    compiler = settings.get_safe("compiler")
-    compiler_version = settings.get_safe("compiler.version")
-    cppstd = settings.get_safe("compiler.cppstd")
+def cppstd_flag(conanfile: "ConanFile") -> str:
```

Comment:
This is not exposed on the external API. A test that uses it with ``from conan.tools.build import cppstd_flag`` should show it.

Reply:
Sorry, forgot to push last changes.